QuizApp/QuizApp/Utils/SecurityStorage.swift

```diff
@@ -0,0 +1,30 @@
+import KeychainAccess
+
+protocol SecurityStorageProtocol {
+
+    func saveAccessToken(accessToken: String)
+
+    func getAccessToken() -> String
```

Comment:
Functions that have no arguments and return a value should be computed properties.
Also, is this contract true? What if there is no access token in the storage?